guideline/cfh.md

```diff
@@ -131,26 +131,11 @@ off by the PAC before they can be put in place.
    Proposals will have to provide details about how these can be implemented at
    the proposed venues.
 
-   1. Food and drinks for lunches and breaks on all conference days (and
```

Comment:
I think food and social events should be moved to "good-to-haves", and we can include how other local or APAC PyCons have been done as a reference. I'm trying to reduce our narrative that imposes restrictions on what _we_ think will make a successful PyCon and leave that to the organizers to decide.

I believe what we should be doing is not to restrict, but to facilitate.